molo/profiles/templates/profiles/register.html

```diff
@@ -2,50 +2,60 @@
+  {% csrf_token %}
+    <fieldset>
+      <label for="id_username">{% trans "Choose a username" %}</label>
+      {{ form.username.errors}}
+      {{ form.username }}
+    </fieldset>
 
-    <form method="post" action="{% url 'molo.profiles:user_register' %}">{% csrf_token %}
-        <p><strong>{% trans "Choose a username" %}</strong></p>
-        {{ form.username.errors}}
-        {{ form.username }}
+    <fieldset>
+      <label for="id_password">{% trans "Choose a 4-digit pin" %}</label>
+      {{ form.password.errors}}
+      {{ form.password }}
+      <p class="sigin-form__input-hint">({% trans "Eg.: 2086" %})</p>
+    </fieldset>
 
-        <p><strong>{% trans "Choose a 4-digit pin" %}</strong></p>
-        {{ form.password.errors}}
-        {{ form.password }}
-        <br>
-        {% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
-        <p><strong>{% trans "Enter your mobile number" %}</strong></p>
-        <p>{% trans "(e.g. +27900000013)" %}</p>
-        {{ form.mobile_number.errors}}
-        {{ form.mobile_number }}
-        <br>
-        {% endif %}
-        {% if settings.profiles.UserProfilesSettings.show_email_field %}
-        <p><strong>{% trans "Enter your email addresss" %}</strong></p>
-        <p>{% trans "(e.g. example@foo.com)" %}</p>
-        {{ form.email.errors}}
-        {{ form.email }}
-        <br>
-        {% endif %}
+    {% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
+    <fieldset>
+      <label for="mobile_number">{% trans "Enter your mobile number" %}</label>
+      {% trans "(e.g. +27900000013)" %}
+      {{ form.mobile_number.errors}}
+      {{ form.mobile_number }}
+    </fieldset>
+    {% endif %}
 
-        {% if form.terms_and_conditions.errors %}
-          <p>{% trans "Please accept the T&Cs in order to complete the registration" %}</p>
-        {% endif %}
-        {{ form.terms_and_conditions }}
-        <span>{% trans "I accept the Terms and Conditions" %}</span></p>
-        <br>
+    {% if settings.profiles.UserProfilesSettings.show_email_field %}
+    <fieldset>
+      <label for="id_email">{% trans "Enter your email addresss" %}</label>
+      {% trans "(e.g. example@foo.com)" %}
+      {{ form.email.errors}}
+      {{ form.email }}
+    </fieldset>
+    {% endif %}
 
-        <input type="submit" class="registerSubmit" value='{% trans "JOIN" %}' />
-        <input type="hidden" name="next" value="{% url 'molo.profiles:registration_done' %}" />
-        <a href="{% url 'molo.profiles:auth_login' %}" class="button inverted">{% trans "already joined?" %}</a>
-    </form>
-  </div>
+    <fieldset>
+      {% if form.terms_and_conditions.errors %}
+      <span class="error">{% trans "Please accept the T&Cs in order to complete the registration" %}</span>
+      {% endif %}
+      {{ form.terms_and_conditions }}
+      <label for="id_terms_and_conditions" class="profiles__terms">{% trans "I accept the Terms and Conditions" %}</label>
```

Comment:
Hmm so this is an error hint for this required checkbox. I think we should give it a class name of .error